tuplex/core/include/physical/PhysicalPlan.h

```diff
@@ -41,7 +41,7 @@ namespace tuplex {
 
         IBackend* backend() const { return _context.backend(); }
 
-        std::shared_ptr<HistoryServerConnector> _hs;
+//        std::shared_ptr<HistoryServerConnector> _hs;
```

Comment:
if this is not needed anymore, can we remove the line?

Reply:
removed.